index.js

```diff
@@ -1,10 +1,29 @@
 const express = require("express");
 const app = express();
-const PORT = process.env.PORT || 4000;
+const authRouter = require("./routers/auth");
+const { PORT } = require("./config/constants");
 const cors = require("cors");
+const loggerMiddleWare = require("morgan");
+const bodyParserMiddleWare = express.json();
+const bookscollectionRouter = require("./routers/bookscollection");
+const userRouter = require("./routers/user");
+const collectionRouter = require("./routers/collection");
+const reviewRouter = require("./routers/review");
```

Comment:
try to maintain an order in the `requires/imports`, normally i do:
1. Libraries (express, axios, react, react-redux, etc)
2. My other files (routers/middlewares in the backend, actions/selectors in FE)
3. Components
4. Css

and after all the imports i start defining consts like `PORT` or `app = express()`